src/System.Drawing.Primitives/tests/SizeFTests.netcoreapp.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using Xunit;
+
+namespace System.Drawing.PrimitivesTest
+{
+    public partial class SizeFTests
+    {
+        #region SizeF * float tests
+
+        private void MultSizeFFloatTester(float width, float height, float multiplier)
+        {
+            SizeF sz = new SizeF(width, height);
+            SizeF mulExpected;
+
+            mulExpected = new SizeF(width * multiplier, height * multiplier);
+
+            Assert.Equal(mulExpected, sz * multiplier);
+            Assert.Equal(mulExpected, multiplier * sz);
+        }
+
+        [Theory]
+        [InlineData(1000.234f, 2400.933f)]
+        [InlineData(float.MaxValue, 0.0f)]
+        [InlineData(float.MaxValue, 1.0f)]
+        [InlineData(float.MaxValue, -1.0f)]
+        [InlineData(float.MaxValue, float.MaxValue)]
+        [InlineData(float.MinValue, 0.0f)]
+        [InlineData(float.MinValue, 1.0f)]
+        [InlineData(float.MinValue, -1.0f)]
+        [InlineData(float.MinValue, float.MinValue)]
+        [InlineData(float.MaxValue, float.MinValue)]
+        [InlineData(float.MinValue, float.MaxValue)]
+        public void MultiplicationTest(float value1, float value2)
+        {
+            MultSizeFFloatTester(value1, 1.0f, value2);
```

Comment:
I think it is fine to have test cases which cover (width, 1f) or (1f, height); not arguing against that. But currently, all of the test cases are of that form. I think that we should at least have a test case which covers (3.0f, 4.0f) * 5.0f, for example.

My suggestion goes further than that, though. If the test case was specified like I mentioned above, I think the readability would also improve greatly. For example, it would be more obvious that `MutliplyTest(3f, 4f, 5f)` would be testing `new SizeF(3f, 4f) * 5f`. Right now I think it is difficult to identify which test parameters end up in where in the assertions, because they are used in a few different combinations.

Does that make sense? Part of this is the subjective "readability" of the test case, so other folks might disagree with my assessment.